src/stories/Badge.stories.js

```diff
@@ -5,6 +5,6 @@ export default {
   title: "Badge"
 };
 
-export const Active = () => <Badge active>Asiatisch</Badge>;
+export const Active = () => <Badge active={true}>Asiatisch</Badge>;
```

Comment:
This change doesn't change the functionality. Actually, it is the same like before. You don't need to change it back, I just want to mention that it is the same.